webtau-core/src/main/java/com/twosigma/webtau/data/render/DataRenderers.java

```diff
@@ -40,6 +40,7 @@ public static String render(Object data) {
         renders.addAll(ServiceUtils.discover(DataRenderer.class));
         renders.add(new TableDataRenderer());
         renders.add(new ToStringRenderer());
+        renders.add(new AnyRenderer());
```

Comment:
One could argue you don't need to actually have an `AnyRenderer` class and just add `Object::toString` to the list but I think that's a stylistic choice more than anything else so happy to leave that up to you.

Reply:
Or `Objects::toString` if you want the null safety though it's not required with the first renderer being the `NullRenderer`.

Reply:
will try to use that instead